hydromt/gis_utils.py

```diff
@@ -303,17 +303,22 @@ def axes_attrs(crs):
     return x_dim, y_dim, x_attrs, y_attrs
 
 
-def meridian_offset(ds, x_name="x", bbox=None):
+def meridian_offset(ds, bbox=None):
     """re-arrange data along x dim."""
-    if ds.raster.crs is None or ds.raster.crs.is_projected:
-        raise ValueError("The method is only applicable to geographic CRS")
+    w, _, e, _ = ds.raster.bounds
+    if (
+        ds.raster.crs is None
+        or ds.raster.crs.is_projected
+        or not np.isclose(e - w, 360)
+    ):
+        raise ValueError("The method is only applicable to global geographic CRS")
```

Comment:
would be nice to mention in the error which f the conditions failed, but this is also acceptable